cni/cmd/istio-cni-repair/main.go

```diff
@@ -182,9 +195,15 @@ func main() {
 		log.Fatalf("Could not construct clientSet: %s", err)
 	}
 
-	podFixer := repair.NewBrokenPodReconciler(clientSet, filters, options.RepairOptions)
+	podFixer := repair.NewBrokenPodReconciler(clientSet, filters, options.RepairOptions, &metrics)
 	logCurrentOptions(&podFixer, options)
 
+	// Start metrics server
+	go func() {
+		http.Handle("/metrics", promhttp.Handler())
+		http.ListenAndServe(":2112", nil)
```

Comment:
If we are picking arbitrary ports I recommend 15014 which has been used by Istiod (and other control planes, prior to monolith) for metrics